src/main/java/org/jetbrains/plugins/spotbugs/actions/HelpAction.java

```diff
@@ -92,13 +92,15 @@ private static StringBuilder createHelpInfo() {
 		ret.append("<h2>").append(VersionManager.getFullVersion()).append("</h2>");
 		ret.append("Website: <a href='").append(VersionManager.getWebsite()).append("'>").append(VersionManager.getWebsite()).append("</a>");
 		ret.append("<br>");
-		ret.append("Download: <a href='").append(VersionManager.getDownloadWebsite()).append("'>").append(VersionManager.getDownloadWebsite()).append("</a>");
-		ret.append("<br>");
-		ret.append("Issue tracker: <a href='").append(VersionManager.getIssueTracker()).append("'>").append(VersionManager.getIssueTracker()).append("</a>");
-		ret.append("<h3>SpotBugs ").append(FindBugsUtil.getFindBugsFullVersion()).append("</h3>");
-		ret.append("Website: <a href='").append(Version.WEBSITE).append("'>").append(Version.WEBSITE).append("</a>");
-		ret.append("<br>");
-		ret.append("Download: <a href='").append(DOWNLOADS_WEBSITE).append("'>").append(DOWNLOADS_WEBSITE).append("</a>");
+		ret.append("Support & Feedback: <a href='mailto:dev@reshiftsecurity.com'>dev@reshiftsecurity.com</a>");
```

Comment:
should we have a support email? is this the support email we are monitoring?

Reply:
Yes, this is the inbox we have for support now.